.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Workflow
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+jobs:
+  php-stan:
+    name: PHP Stan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+
+      - name: Configure
+        run: composer config repositories.repo.magento.com composer https://repo.magento.com && composer config -g http-basic.repo.magento.com ${{ secrets.REPO_PUB_KEY }} ${{ secrets.REPO_SEC_KEY }}
```

Comment:
@aashwin-rvvup this seems to be a pipeline for the magento module instead of the SDK?